Source/Scene/GltfLoaderUtil.js

```diff
@@ -74,6 +75,7 @@ GltfLoaderUtil.createSampler = function (options) {
   options = defaultValue(options, defaultValue.EMPTY_OBJECT);
   var gltf = options.gltf;
   var textureInfo = options.textureInfo;
+  var compressedTextureNoMipmap = options.compressedTextureNoMipmap;
```

Comment:
My idea was that whoever calls `createSampler` would follow the logic: 

```js
if (textureIsCompressed && textureHasNoMipmap) {
  // make a note to pass `compressedTextureNoMipmap = true` to createSampler()
}
```

Reply:
You can add code like that to `GltfTextureLoader` now. It should already know whether the texture is compressed and whether it doesn't have a mipmap.